samples/nrf9160/nrf_cloud_rest_device_message/src/main.c

```diff
@@ -344,6 +346,47 @@ static int setup(void)
 	return 0;
 }
 
+int send_hello_world_msg(void)
+{
+	int err;
+	cJSON *msg_obj = NULL;
+	int64_t time_now;
+
+	/* Send off a hello world message! */
+	/* This does not match a predefined schema. But that's not a problem! */
+	msg_obj = cJSON_CreateObject();
```

Comment:
Instead of introducing cJSON, you can do something like:

```
#define SAMPLE_MSG "{\"sample_message\":" \
			"\"Hello World, from the REST Device Message Sample!\", " \
                         "\"ts\": %ld}\""
#define SAMPLE_MSG_BUF_SIZE (sizeof(SAMPLE_MSG) + 19)

char buf[SAMPLE_MSG_BUF_SIZE];
    
snprintf(buf, SAMPLE_MSG_BUF_SIZE, SAMPLE_MSG, timestamp);
```

otherwise you will need to delete msg_obj and free the buffer generated by `cJSON_PrintUnformatted`